src/components/experiments/ExperimentTable.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { List } from "immutable";
+
+export class ExperimentTable extends React.Component {
+  render() {
+    return (
+      <div>
```

Comment:
An alternative to littering the DOM with useless `div` elements that only exist to satisfy Reacts need for a single parent element you could just use:
```jsx
<React.Fragment>
  <Table>...</Table>
</React.Fragment>
```

Reply:
Although in this case I'm not sure you need the `div` or `Fragment` at all since `table` has no siblings.

Reply:
Oh you're right, it runs fine without it! 🎉